Week1/solution4.py

```diff
@@ -0,0 +1,18 @@
+'''
+Write a python script to enter any number, if it is integer number, then check its armstrong or not.
+Print appropriate message if it is not palindrome.
+'''
+n=float(input("Enter a number:"))

```

Comment:
why n is float?

Reply:
float datatype was used so variable can take both float or int value which is needed to check condition, if number is integer as per question.